lib/fpm/command.rb

```diff
@@ -3,7 +3,6 @@
 require "fpm/version"
 require "fpm/util"
 require "clamp"
-require "ostruct"
```

Comment:
I added this to get rid of a Ruby warning about a future change: "warning: ostruct was loaded from the standard library, but will no longer be part of the default gems starting from Ruby 3.5.0."